modules/standard/OS.chpl

```diff
@@ -366,24 +373,69 @@ module OS {
     //
     // sys/stat.h
     //
-    extern const S_IRWXU: mode_t;
-    extern const S_IRUSR: mode_t;
-    extern const S_IWUSR: mode_t;
-    extern const S_IXUSR: mode_t;
-
-    extern const S_IRWXG: mode_t;
-    extern const S_IRGRP: mode_t;
-    extern const S_IWGRP: mode_t;
-    extern const S_IXGRP: mode_t;
-
-    extern const S_IRWXO: mode_t;
-    extern const S_IROTH: mode_t;
-    extern const S_IWOTH: mode_t;
-    extern const S_IXOTH: mode_t;
-
-    extern const S_ISUID: mode_t;
-    extern const S_ISGID: mode_t;
-    extern const S_ISVTX: mode_t;
+    inline proc S_IRWXU:mode_t {
+      extern proc chpl_os_posix_S_IRWXU():mode_t;
+      return chpl_os_posix_S_IRWXU();
+    }
+    inline proc S_IRUSR:mode_t {
```

Comment:
I would think we would only need these for the multi-bit ones and not for the single-bit ones which are (should be?) `S_I{R,W,X}{USR,GRP,OTH}`.  That said, one could argue that in saying this I am advocating coding to an implementation rather than the interface, which is the opposite of what I always say is the right thing to do. So doing them all is probably better.